GENERATOR-2021 Added generator

kairoslav opened the pull request.

kairoslav committed:
Add: MD Generator

Vivelapaix commented on leetcode/MDGenerator/LeetCodeSourceData.java:
```diff
@@ -0,0 +1,83 @@
+package com.company;
```
Your Pull Request title: Add: MD Generator

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']
* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']

kairoslav committed:
Moved: the generator is moved to src direction;

kairoslav committed:
Added: "stream" instead of "for" in data processing

kairoslav committed:
Reformat: String.format() is changed to StringBuilder.append()

//https://redfin.engineering/java-string-concatenation-which-way-is-best-8f590a7d22a8

kairoslav committed:
Reformat: beautification is applied

kairoslav committed:
Fix: first letter upper

kairoslav committed:
MDGenerator was moved to another directory
